src/Elmish.WPF/BindingData.fs

```diff
@@ -541,6 +474,22 @@ module internal BindingData =
 
   module SubModel =
 
+    let mapMinorTypes
+        (outMapBindingModel: 'bindingModel -> 'bindingModel0)
+        (outMapBindingMsg: 'bindingMsg -> 'bindingMsg0)
+        (outMapBindingViewModel: 'bindingViewModel -> 'bindingViewModel0)
+        (inMapBindingModel: 'bindingModel0 -> 'bindingModel)
+        (inMapBindingMsg: 'bindingMsg0 -> 'bindingMsg)
+        (inMapBindingViewModel: 'bindingViewModel0 -> 'bindingViewModel)
+        (d: SubModelData<'model, 'msg, 'bindingModel, 'bindingMsg, 'bindingViewModel>) = {
+      GetModel = d.GetModel >> ValueOption.map outMapBindingModel
+      CreateViewModel = fun args -> d.CreateViewModel(args |> ViewModelArgs.map inMapBindingModel outMapBindingMsg) |> outMapBindingViewModel
```

Comment:
```suggestion
      CreateViewModel =
        fun args ->
          d.CreateViewModel(args |> ViewModelArgs.map inMapBindingModel outMapBindingMsg) |> outMapBindingViewModel
```

Reply:
I'd prefer not to make formatting adjustments at this time

Reply:
I'd prefer not to make formatting adjustments at this time